src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -197,7 +197,13 @@ private static void InitTelemetry()
             Logger.AddOrUpdateContextProperty(TelemetryProperty.Version, VersionTools.GetAppVersion());
             Logger.AddOrUpdateContextProperty(TelemetryProperty.AppSessionID, Guid.NewGuid().ToString());
             Logger.AddOrUpdateContextProperty(TelemetryProperty.SessionType, "Desktop");
-            Logger.AddOrUpdateContextProperty(TelemetryProperty.ReleaseChannel, ConfigurationManager.GetDefaultInstance().AppConfig.ReleaseChannel.ToString());
+            Logger.AddOrUpdateContextProperty(TelemetryProperty.WindowsVersion, ConfigurationManager.GetDefaultInstance().AppConfig.ReleaseChannel.ToString());
```

Comment:
Looks like TelemetryProperty.WindowsVersion and TelemetryProperty.ReleaseChannel got flipped between here and line 205
```suggestion
            Logger.AddOrUpdateContextProperty(TelemetryProperty.ReleaseChannel, ConfigurationManager.GetDefaultInstance().AppConfig.ReleaseChannel.ToString());
```